docs/references/api/resource_representation.rst

```diff
@@ -16,6 +16,30 @@ Use the Accept request header to specify the acceptable format (or formats) for
   curl "http://localhost:3000/people" \
     -H "Accept: application/json"
 
+.. note::
+  
+  The ordering of columns in the response isn't guaranteed to align with the order specified in the ``select`` clause. For example, with resource embedding:
+
+  .. code-block:: bash
+    
+    http://localhost:3000/films?select=directors(last_name,id),title
+  
+  We may get:
+
+  .. code-block:: bash
+
+    [
+      {
+        "title": "title",
+        "directors": {
+          "id": 5,
+          "last_name": "name"
+        }
+      }
+    ]
+
+  See: `github comment <https://github.com/PostgREST/postgrest/issues/3962#issuecomment-2748441060>`_ for why the ordering is not enforced.
```

Comment:
It would be better to inline this explanation:

```suggestion
This is in line with the [JSON schema spec](https://json-schema.org/draft/2020-12/json-schema-core):

> JSON Schema is defined over "application/json" or compatible documents, including media types with the "+json" structured syntax suffix.
```

Note: this is not valid rst, needs to be edited.